overlays/speakeasy-modifications-overlay.yaml

```diff
@@ -365,976 +356,987 @@ actions:
     x-speakeasy-metadata:
       after: sdk.displayableLists.update()
       before: sdk.Displayable Lists.updatedisplayablelists()
-      created_at: 1743113936039
-      reviewed_at: 1743114533812
+      created_at: 1745500722403
+      reviewed_at: 1745505888484
       type: method-name
-  - target: $["paths"]["/rest/api/v1/createanonymoustoken"]["post"]
+  - target: $["paths"]["/rest/api/v1/listshortcuts"]["post"]
     update:
-      x-speakeasy-name-override: createAnonymousToken
-      x-speakeasy-group: client.authentication
+      x-speakeasy-group: client.shortcuts
+      x-speakeasy-name-override: list
     x-speakeasy-metadata:
-      after: sdk.authentication.createAnonymousToken()
-      before: sdk.Authentication.createanonymoustoken()
-      created_at: 1743113936039
-      reviewed_at: 1743114533812
+      after: sdk.shortcuts.list()
+      before: sdk.Shortcuts.listshortcuts()
+      created_at: 1745500722403
+      reviewed_at: 1745505888484
       type: method-name
-  - target: $["paths"]["/rest/api/v1/getanswerboard"]["post"]
+  - target: $["paths"]["/rest/api/v1/deleteannouncement"]["post"]
     update:
-      x-speakeasy-name-override: getBoard
-      x-speakeasy-group: client.answers
+      x-speakeasy-group: client.announcements
+      x-speakeasy-name-override: delete
     x-speakeasy-metadata:
-      after: sdk.answers.getBoard()
-      before: sdk.Answers.getanswerboard()
-      created_at: 1743113936039
-      reviewed_at: 1743114533812
+      after: sdk.announcements.delete()
+      before: sdk.Announcements.deleteannouncement()
+      created_at: 1745500722403
+      reviewed_at: 1745505888484
       type: method-name
-  - target: $["paths"]["/rest/api/v1/createdisplayablelists"]["post"]
+  - target: $["paths"]["/rest/api/v1/deleteshortcut"]["post"]
     update:
-      x-speakeasy-group: client.displayableLists
-      x-speakeasy-name-override: create
+      x-speakeasy-group: client.shortcuts
+      x-speakeasy-name-override: delete
     x-speakeasy-metadata:
-      after: sdk.displayableLists.create()
-      before: sdk.Displayable Lists.createdisplayablelists()
-      created_at: 1743113936039
-      reviewed_at: 1743114533812
+      after: sdk.shortcuts.delete()
+      before: sdk.Shortcuts.deleteshortcut()
+      created_at: 1745500722403
+      reviewed_at: 1745505888484
       type: method-name
-  - target: $["paths"]["/rest/api/v1/editanswerboard"]["post"]
+  - target: $["paths"]["/rest/api/v1/previewannouncementdraft"]["post"]
     update:
-      x-speakeasy-name-override: updateBoard
-      x-speakeasy-group: client.answers
+      x-speakeasy-group: client.announcements
+      x-speakeasy-name-override: previewDraft
+      operationId: previewAnnouncementDraft
```

Comment:
Note that `x-speakeasy-group` and `x-speakeasy-name-override` control the method name. However, changing the `operationId` impacts the names of the request and response bodies if they're inline. So this method went from

Before
```
sdk.previewannouncementdraft(request: PreviewannouncementdraftRequest): PreviewannouncementdraftResponse
```

After
```
sdk.client.announncements.previewDraft(request: PreviewAnnouncementDraftRequest): PreviewAnnouncementDraftResponse
```